addons/hr_expense/models/hr_expense.py

```diff
@@ -1142,6 +1142,7 @@ def _do_approve(self):
             'next': {'type': 'ir.actions.act_window_close'},
         })
 
+        self.approval_date = fields.Datetime.now()
```

Comment:
I see in 16.2, we're using `'approval_date': fields.Date.context_today(sheet)`, and only in `filtered_sheet` defined above. Shouldn't we do the same if 16.2 works as we want?

Reply:
Yes you are right, don't know how i missed that :thinking:, will change this today ! Thanks !